test/thrift/parser_test.exs

```diff
@@ -1,8 +1,10 @@
 defmodule ParserTest do
-
   use ExUnit.Case, async: true
```

Comment:
I don't think this can be async anymore if we share the test file directory across individual tests.